src/http/app.ts

```diff
@@ -156,12 +186,18 @@ app.get("/data/:chainId/rounds/:roundId/vote_coefficients.csv", async (req, res)
       fs.promises.readFile("./data/passport_scores.json", { encoding: "utf8", flag: "r" }),
     ]);
 
-    const passportScores = JSON.parse(data);
+    const passportScores: FullPassportScore[] = JSON.parse(data);
     const passportScoresMap = processPassportScores(passportScores);
 
     const records = votes.map((vote: any) => {
       const voter = vote.voter.toLowerCase();
-      const combinedVote = { ...vote, ...passportScoresMap[voter] ?? {} };
+      const score = passportScoresMap[voter];
+      const coefficient = score !== undefined ? 1 : 0;
+      const combinedVote = {
+      ...vote,
+      ...(score ?? {}),
+      coefficient,
```

Comment:
score already has a coefficient right? this would be overriding it

Reply:
but in the case that the score is not found in the passport file we want to show 0 as coefficient

Reply:
yes I'll push in a second because I changed only the firs part

Reply:
I changed it back to draft